testnet/stacks-node/src/main.rs

```diff
@@ -100,19 +100,7 @@ fn main() {
             args.finish().unwrap();
             ConfigFile::helium()
         }
-        "neon" => {
-            args.finish().unwrap();
-            ConfigFile::neon()
-        }
-        "argon" => {
-            args.finish().unwrap();
-            ConfigFile::argon()
-        }
-        "krypton" => {
-            args.finish().unwrap();
-            ConfigFile::krypton()
-        }
-        "xenon" => {
+        "testnet" => {
```

Comment:
Can delete these functions in `testnet/stacks-node/src/config.rs` ( neon, argon, krypton)

Reply:
Good idea, addressed with latest